### DISCUSSION TITLE
Friends or Following Activity Feed

### DISCUSSION THREAD

**Discussion by kevincador** [trakt/trakt-api#469] [Q&A]
Hey,

Would it be possible to have an endpoint exposed to fetch a following activity feed or a friends activity feed?

I guess it could be done with a lot of smaller calls on each friends/following but depending on the list it could hit hard. 

I see the website and app have those feeds. It would be great to have more activities tho, because it seems only the watch activities are displayed. I could be wrong but I don't see ratings, comments,... Anyway, the more the better 🤗

Thanks!!

Kevin

**Comment by rectifyer** [comment]
Planning to add this in the future. The problem right now is performance. I plan to cache things better so a full social feed can be done much quicker, but I have no ETA on this just yet.
